src/services/BudgetService.ts

```diff
@@ -183,4 +188,65 @@ export class BudgetService {
       await QuarterBudgetModel.updateBudget(budget)
     }
   }
+
+  static getUncappedRoundedPercentage(value: number, total: number): number {
+    return Math.round((value * 100) / total)
+  }
+
+  static async getExpectedAllocatedBudget() {
+    const activeGrantProposals = await ProposalService.getActiveGrantProposals()
+    const categories = Object.values(NewGrantCategory).map(snakeCase)
+    const currentBudget = await this.getCurrentBudget()
+    const expectedBudget: ExpectedBudget = this.initializeExpectedBudget(categories, currentBudget)
+    this.addContestants(activeGrantProposals, expectedBudget)
```

Comment:
I still don't like functions that change objects inside them without returning anything... 😞  I think they make code harder to read and to follow.